### ISSUE TITLE
✨ Use ∞ instead of inf in excel parameter files

### ISSUE THREAD

**Issue by s-weigand** [glotaran/pyglotaran#888]
### Is your feature request related to a problem? Please describe.

Excel is missing the concept of infinity, and thus when loading a `csv` parameter file `inf` gets interpreted as a string, and `-inf` is displayed as an error in the columns `minimum`/`maximum`.

### Describe the solution you'd like

For excel `IO` (#844) we should convert the columns `minimum`/`maximum` to strings and replace `inf` -> `∞` and `-inf` -> `-∞` before writing. 
For reading, we should reverse the replacing and convert the columns back to floats.
See the [docs of pandas on string manipulation](https://pandas.pydata.org/pandas-docs/stable/user_guide/text.html).

Since this is an excel issue IMHO it should only be done for the excel IO.

### Describe alternatives you've considered

Alternatively, the converting back could also be done in `ParameterGroup.from_dataframe`, where would first check if the `dtype` of the column is a string. 

https://github.com/glotaran/pyglotaran/blob/d05c042a6e4dffeac556d117f30a7b52f4e2b98f/glotaran/parameter/parameter_group.py#L145

That way we could add an `excel_compatible` option to `save_parameters`, so users can view their `csv` files in excel.
https://github.com/glotaran/pyglotaran/blob/d05c042a6e4dffeac556d117f30a7b52f4e2b98f/glotaran/builtin/io/csv/csv.py#L16

**Comment by joernweissenborn** [comment]
I am strongly opting for creating a separate excel IO plugin for that.  It can then be specified via the `SavingOptions` to output the parameters as `xlsx` directly when parameter format is set to `excel`. We should show this more prominently in our docs and guides.

I am strongly against using unicode symbols in csv. Although modern things can handle it, we should try to restrict csv to ASCII for the sake of compatibility.

This could be solved alongside #844

**Comment by jsnel** [comment]
Why not simply allow "", as in nothing or <blank>, an empty string, if you will? 

So instead of changing
```csv
,minimum,maximum,
,-inf,+inf,
```
to
```csv
,minimum,maximum,
,-∞,+∞,
```
just allow for
```csv
,minimum,maximum,
,,,
```

This is more intuitive to many users than anything else. Also, when editing a csv in a spreadsheet application that is equivalent to just clearing/blanking the cells.

A blank cell for minimum/maximum pretty unambiguously implies no minimum/maximum bound.

**Comment by s-weigand** [comment]
I guess using `-∞,+∞` vs. `,` comes down to verbosity and how good we communicate to the users what the default values are.
So if we want to go with `,` we should have a cheat sheet which actually is a cheat sheet, so users can quickly lookup default values (which we should have in any case).

So from me, there is a +1 for `,` but for that case, we need to fill the nan values correspondingly, which best should be done in `Parameter` itself.
```python
In [1]: import pandas as pd

In [2]: import numpy as np

In [3]: from io import StringIO

In [4]: foo = pd.read_csv(StringIO("minimum,maximum\n,\n1,\n,0.2"))

In [5]: foo
Out[5]:
   minimum  maximum
0      NaN      NaN
1      1.0      NaN
2      NaN      0.2

In [6]: foo["minimum"].fillna(-np.inf, inplace=True)

In [7]: foo["maximum"].fillna(np.inf, inplace=True)

In [8]: foo
Out[8]:
   minimum  maximum
0     -inf      inf
1      1.0      inf
2     -inf      0.2
```

**Comment by s-weigand** [comment]
Closed in #896 with `replace_infinfinity=True`
